### ISSUE TITLE
Reserves toolbar text is black on brown

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#1574]
With the new dark background for toolbars, course reserves ended up with an inaccessible text/background contrast. Need to change text from black to white. Adding `color: white` to `.table-top-bar` seems to get all the text bits.
https://sw-webapp-sandbox-f.stanford.edu/reserves

Before
![screen shot 2017-08-10 at 11 07 41 am](https://user-images.githubusercontent.com/6945691/29184812-5bb701c8-7dbc-11e7-9ef9-b511b9ff7f6b.png)

After
![screen shot 2017-08-10 at 11 07 29 am](https://user-images.githubusercontent.com/6945691/29184821-6224e6c4-7dbc-11e7-8d53-692e6917dd8b.png)
